lib/src/component_declaration/flux_component.dart

```diff
@@ -27,9 +27,64 @@ import 'annotations.dart' as annotations;
 import 'builder_helpers.dart';
 import 'component_base_2.dart';
 
-/// Builds on top of [UiProps], adding typed props for [Action]s and [Store]s in order to integrate with w_flux.
+part 'flux_component.over_react.g.dart';
+
+/// Builds on top of [UiProps], adding typed [Action] and [Store] props for use with [FluxUiComponent2].
+///
+/// __Example:__
+///
+/// ```dart
+/// class YourComponentProps extends UiProps with FluxUiPropsMixin<YourFluxActionsClass, YourFluxStoreClass> {
+///   // Additional arbitrary props for your component can go here
+/// }
+/// ```
+mixin FluxUiPropsMixin<ActionsT, StoresT> on UiProps implements FluxUiProps<ActionsT, StoresT> {
+  @override
+  String get _actionsPropKey => '${propKeyNamespace}actions';
```

Comment:
Hmmm I wonder if we should just have these be normal props.

`propKeyNamespace` is going to be `''` for all new mixin-based props classes, and I think it'd be less 
confusing if this behaved like a normal mixin.

I've also been wanting to deprecate `propKeyNamespace` since it really doesn't serve any other purpose, but had forgotten about it.

If there are no objections, I'll just address this in my branch.

Reply:
@aaronlademann-wf @joebingham-wk @sydneyjodon-wk @kealjones-wk Let me know your thoughts on this.

If you're cool, please give a thumbs up reaction. If not, say why. If you don't have an opinion, do another reaction of your choice 😄.

Reply:
Looks like there's only one real usage, which would be improved via metFor(FluxUiPropsMixin)
> ```dart
> @override
> Iterable<ConsumedProps> get consumedProps => [
>   GraphDataProps.meta,
>   GraphDataPropsMixin.meta,
>   SingleEdgeFormItemProps.meta,
>    
>   /// `FluxUiProps` is not a generated props class so the keys are not removed when forwarding.
>   ConsumedProps([PropDescriptor('${props.propKeyNamespace}actions')], ['${props.propKeyNamespace}actions']),
>   ConsumedProps([PropDescriptor('${props.propKeyNamespace}store')], ['${props.propKeyNamespace}store']),
> ];